src/applications/personalization/profile/tests/e2e/direct-deposit/update-flow.cypress.spec.js

```diff
@@ -117,4 +133,34 @@ describe('Direct Deposit', () => {
       cy.axeCheck();
     });
   });
+  describe('when editing both at the same time and they both fail to update', () => {
```

Comment:
This new test makes sure we fixed the a11y issues this PR was attempting to fix. Namely: with both edit bank info forms visible, does it pass an aXe scan? And also that when both form's error alerts are shown, does it pass an aXe scan?